src/styles/nav.scss

```diff
@@ -0,0 +1,79 @@
+@import "../utils/variables.scss";
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
```

Comment:
1. it's better not reset styles at * selector. Only use box-sizing: border-box for this selector
2. move these styles to reset.scss